src/main.cpp

```diff
@@ -33,19 +33,18 @@ class $modify(CCKeyboardDispatcher) {
     }
 };
 
-#ifdef GEODE_IS_MOBILE
-
 #include <Geode/loader/GameEvent.hpp>
 $execute {
     new EventListener<GameEventFilter>(+[](GameEvent*) {
-        DragButton::get();
+        GEODE_DESKTOP(if (DevTools::get()->getSettings().buttonEnabled)) DevTools::get()->setupDragButton();
```

Comment:
maybe make a devtools method isButtonEnabled that always returns true on mobile, instead of having GEODE_DESKTOP everywhere